profiles/fscloud/README.md

```diff
@@ -1,11 +1,91 @@
-# Financial Services Cloud Profile
+# Profile for IBM Cloud Framework for Financial Services
 
-This is a profile for COS that meets FS Cloud requirements. This profile assumes you are deploying into an already compliant account.
-It has been scanned by [IBM Code Risk Analyzer (CRA)](https://cloud.ibm.com/docs/code-risk-analyzer-cli-plugin?topic=code-risk-analyzer-cli-plugin-cra-cli-plugin#terraform-command) and meets all applicable goals with the following exceptions:
-- 3000107 - Check whether Cloud Object Storage network access is restricted to a specific IP range
-  - This is ignored because the CBR locks this down and CRA does not check this
-- 3000116 - Check whether Cloud Object Storage bucket resiliency is set to cross region
-  - This is ignored because cross-regional buckets and KYOK encryption (HPCS) are not compatible. The solution is to provision two buckets with replication in separate regions. This gives us
-    cross region support and KYOK encryption but CRA does not pick this up.
+This code is a version of the [parent root module](../../) that includes a default configuration that complies with the relevant controls from the [IBM Cloud Framework for Financial Services](https://cloud.ibm.com/docs/framework-financial-services?topic=framework-financial-services-about). See the [Example for IBM Cloud Framework for Financial Services](/examples/fscloud/) for logic that uses this module. The profile assumes you are deploying into an account that is in compliance with the framework.
 
-## Note: If no Context Based Restriction(CBR) rules are passed, you must configure Context Based Restrictions externally to be compliant.
+The default values in this profile were scanned by [IBM Code Risk Analyzer (CRA)](https://cloud.ibm.com/docs/code-risk-analyzer-cli-plugin?topic=code-risk-analyzer-cli-plugin-cra-cli-plugin#terraform-command) for compliance with the IBM Cloud Framework for Financial Services profile that is specified by the IBM Security and Compliance Center. The scan passed for all applicable goals with the following exceptions:
+
+> 3000107 - Check whether Cloud Object Storage network access is restricted to a specific IP range
+
+The rule is ignored because it is covered by the context-based restriction rule. CRA does not check the rule.
+
+> 3000116 - Check whether Cloud Object Storage bucket resiliency is set to cross region
+
+This is ignored because cross-regional buckets and keep-your-own-key (KYOK) encryption (HPCS) are not compatible. The solution is to provision two buckets with replication in separate regions. This solution gives you cross-region support and KYOK encryption. CRA does not validate the rule.
```

Comment:
I don't think this is true. According to this [doc](https://cloud.ibm.com/docs/cloud-object-storage?topic=cloud-object-storage-hpcs):

> only Regional and US Cross Region are currently supported

which got me thinking, we probably need to update the FSCloud profile to handle cross regional buckets. Thoughts @shemau ?

In the meantime, @SirSpidey you can update this text with something like this:

This is ignored because the current FSCloud module provisions two regional buckets with replication in separate regions in order to achieve the same resiliency as cross regional buckets. CRA does not currently support checking for replication rules yet, so hence the above failure.